### DISCUSSION TITLE
关于n-select在n-form中当select的options配置的value值为数字时，验证规则不通过

### DISCUSSION THREAD

**Discussion by 996wuxian** [tusen-ai/naive-ui#6209] [General]
错误格式：
{ label: '不限', value: 0 },
{ label: '男', value: 1 },
{ label: '女', value: 2 }
正确格式：
{ label: '不限', value: '0' },
{ label: '男', value: '1' },
{ label: '女', value: '2' }
为啥数字不能捏？是在这个过程，转换了吗还是

**Comment by clueless328** [comment]
![image](https://github.com/user-attachments/assets/dbb88175-6a8f-41df-bd75-dbfefdeb1e5e)
我也遇到了，这种value是数字的，你的rule还得补上type

**Comment by jahnli** [comment]
`type: number`
